### PR TITLE
"Support for WebWorker." Fixing missing self definition which fails linting

### PR DESCRIPTION
**Pull request by dryajov**
This fixes linting issues in PR #59

### CODE REVIEW

**Comment by dryajov** [comment]
@diasdavid @dignifiedquire Could we merge this in? I'd love to get this in master so we can get it running in ServiceWorkers.

**Comment by dryajov** [comment]
It seems like it would be better adding WebWorker tests to all the js projects... Its possible to do that with a karma plugin like this one - https://github.com/Joris-van-der-Wel/karma-mocha-webworker. I can bolt it as a one off to this project, but it'd be a hack. 

Not sure how you guys go about making this sort of decisions, but I'd be happy to help with anything I can. 

I've got a PR (https://github.com/dignifiedquire/aegir/pull/94) in aegir that addresses this issue. It's a POC, but seems to work as intended on this project at least.

**Comment by dignifiedquire** [comment]
@dryajov thank you, lets get the aegir PR shipped and then we can ship this.

**Comment by dryajov** [comment]
@diasdavid @dignifiedquire Can we get this PR merged - https://github.com/dignifiedquire/aegir/pull/94 (unless there is something else outstanding) so we can merge this changes in? 

Also, we should be able to move the rest of the projects to running tests in WebWorkers once that's in, not sure how you want to track that effort.

**Comment by dignifiedquire** [comment]
Thanks, will run this through in the morning and hopefully ship then. 

Let's open an issue on ipfs/js-ipfs for tracking.

File: package.json [hunk 1]
```diff
@@ -75,7 +76,9 @@
     "Friedel Ziegelmayer <dignifiedquire@gmail.com>",
     "Greenkeeper <support@greenkeeper.io>",
     "Richard Littauer <richard.littauer@gmail.com>",
+    "Tom Swindell <t.swindell@rubyx.co.uk>",
     "Yusef Napora <yusef@napora.org>",
+    "dryajov <dryajov@gmail.com>",
```
Comment (dignifiedquire) [inline package.json]: Can you drop these two contributor additions, they are auto generated on release

File: package.json [hunk 2]
```diff
@@ -14,10 +14,11 @@
   "scripts": {
     "lint": "aegir-lint",
     "build": "aegir-build",
-    "test": "npm run test:node && npm run test:no-webcrypto && npm run test:browser",
+    "test": "npm run test:node && npm run test:no-webcrypto && npm run test:browser && npm run test:webworker",
     "test:node": "aegir-test --env node",
     "test:no-webcrypto": "NO_WEBCRYPTO=true aegir-test --env node",
     "test:browser": "aegir-test --env browser",
+    "test:webworker": "aegir-test --env webworker",
```
Comment (dignifiedquire) [inline package.json]: Need the `--env webworker` for the releases as well

Comment (dryajov) [inline package.json]: Is that needed? Looking at `aegir`, the tests tasks already get invoked on release and I've added the `test:webworker` task there.

**Comment by dignifiedquire** [comment]
Moved to #63

**Comment by dignifiedquire** [comment]
Just released :)

**Pull request closed without merging**